gnucash/import-export/csv-imp/csv-account-import.c

```diff
@@ -78,7 +78,8 @@ fill_model_with_match(GMatchInfo *match_info,
  * Parse the file for a correctly formatted file
  *******************************************************/
 csv_import_result
-csv_import_read_file (const gchar *filename, const gchar *parser_regexp,
+csv_import_read_file (GtkWindow *win, const gchar *filename,
```

Comment:
An oversight probably: you've added a parent widget in the function call but as far as I can see you're not using it ?